### PR TITLE
Scott day

### PR DESCRIPTION
**Pull request by Caution2TheWind**

### CODE REVIEW

**Review by myhe01 — APPROVED** [review]
LGTM

File: design-details/sensing.tex [hunk 1]
```diff
@@ -6,65 +6,101 @@ \subsection{Sensing Subsystem}
...
 
-That’s right, the first part of the optical subsystem is the soil. Structures in the soil emit trace electromagnetic signals that correspond to the soil matrix, soil temperature, soil moisture, and chemical content of the soil. (Link papers)
-In order to ensure sufficient signal to noise ratio, the dirt is going to have to be probed by a bright, broadband source covering the visible and Near Infrared regime. Halogen lamps will do (link papers) but it may be possible to achieve the same effect with LEDs near the surface. 
-An additional concern is that irregularities in the soil topography may reflect back light that would otherwise indicate organic components or water. In order to counter this, it is going to be necessary to flatten the dirt with a stamp or roller.
+\begin{figure}[H]
+    \caption{Spectral Output of a Tungsten Lamp}
+    \centering
+    \includegraphics[width=0.75\textwidth]{images/TungstenLamp.jpg}
+\end{figure}
+Reference Link: https://zeiss-campus.magnet.fsu.edu/articles/lightsources/tungstenhalogen.html
```
Comment (NChitty) [inline design-details/sensing.tex:28]: Please remove all the reference links, and just send them to me in an excel spreadsheet or similar and I can create the references at the end of the document

File: design-details/sensing.tex [hunk 2]
```diff
@@ -6,65 +6,101 @@ \subsection{Sensing Subsystem}
...
 
-\paragraph{LEDs}
+\begin{figure}[H]
+    \caption{Fiber Position in Soil Drill}
+    \centering
+    \includegraphics[width=0.75\textwidth]{images/Acrylic Block Fiber positioner.png}
+\end{figure}
+Reference paper titled: On line measurement of some selected soil properties using a VIS NIR sensor
```
Comment (NChitty) [inline design-details/sensing.tex:36]: Remove references like this and just send me the link or reference formatted into title, author, date, etc.

File: design-details/sensing.tex [hunk 3]
```diff
@@ -6,65 +6,101 @@ \subsection{Sensing Subsystem}
...
+\end{figure}
 
-    The grating will be a 1200 groove per mm aluminum coating 1000nm blazed reflection grating. 
- 
-The grating equation governs the angle at which light of different frequencies will reflect. 
-The first three orders of diffraction are plotted below for a 1200 groove per mm grating. The polar plot is degrees, ranging from 400nm to 1700nm. Rho represents the order of diffraction.
-An angle of incidence of 45 degrees will produce backreflection toward the fiber collimator, however an angle of incidence at 50 degrees will not.
+The beam coming out of the fiber will have a nonzero width. When the 1.7mm diameter beam hits the surface of the grating, since the grating is at 45 degrees from the beam, light will be propagating from an area with a diameter larger than 1.7mm.
 
+(Placeholder: trig equation and image)
```
Comment (NChitty) [inline design-details/sensing.tex:68]: I can't approve this PR with placeholders inside

File: design-details/sensing.tex [hunk 4]
```diff
@@ -6,65 +6,101 @@ \subsection{Sensing Subsystem}
...
 
-\paragraph{Carriage driven Photodiodes}
+\begin{figure}[H]
+    \caption{Basic Ray Trace}
+    \centering
+    \includegraphics[width=0.75\textwidth]{images/BasicRayTrace.png}
+\end{figure}
+Reference Link: https://www.newport.com/n/fiber-optic-coupling
```
Comment (NChitty) [inline design-details/sensing.tex:84]: reference

File: images/Untitled Diagram.drawio [hunk 5]
```diff
@@ -0,0 +1 @@
... version="20.3.0" type="device"><diagram id="sM39jpTHVHrNeqXCZMDn" name="Page-1">7ZnbcpswEIafxpftgMTBuWwcO53WnaTjdtxcZRSjAg0gKkRs+vQVIBCYsQcfatFMcuGgZSWk/T/W0noEJ+HmlqLY+0IcHIyA5mxG8GYEgK5rgP/LLVlpMW1hcKnvCCdpWPh/sDBqwpr6Dk5ajoyQgPlx27giUYRXrGVDlJJ12+0nCdpPjZGLO4bFCgVd69J3mFdax6Ym7R+x73qsXrC4E6LKWRgSDzlk3TDB6QhOKCGsvAo3ExzkwaviUvab7bhbT4ziiPXp8Pz58Wrq/E7A4+2nb19T7c6By3e6KSbHsmrF2OEBEE1CmUdcEqFgKq3XlKSRg/NhNd6SPnNCYm7UufEXZiwTaqKUEW7yWBiIu3jjsx959/fAFM2Hxq2bjRi6aGRVI2I0K3uZVfOheU92K1pVv3KB+ap2Bk6YEpLSFd4XLQEgoi5me/xgLS9/LzAJMZ8P70dxgJj/0p4HEoC6tZ/UkF8IGQ+QFACVkuoNPYVQPRRt6Snl3aFowih5xhMSEFqsD+Z/s5karW2VWotxX1CQiifdU7LCScIjsw2BlDjXaO35DC9iVMRgzXN3W04xLqYMb/aHsrt00QFeicQnMr8lmmuZRvUqN3qNFFr5nT9Yr/7FOCP+oCf+htJU18F/jhI8APQ1s4U+hMrZ1ztBeWN/79d3D/aBSvZhh32++U75kweQ+nWjnfoHwD984783/0ZP/k2V/Bsd/hcojLluquE3rMHBb6iEXwJfn9b6nPHs1iFPvxz9Zk/6d1BwGfrNLv04GsKu39KHRr/dCdUIWAF/6rXjv/BLN7+8S1mcMu628F0OeVFl4h/fi92klsdVWyKWN2bzYs7cM31KsoThsBruiVajVRbu3XiGem1gW5taB3WZyVabmY4rP9kH1Z861YrZLK9XnDdpjXsmrVMrU0XXD5SirOEQEz9iSWPk+9zQ2A5uVQKAtlWp3PIHcK8/vyhnILmrl3ICiuPhfEn+m0LooEA89ex0FIh1Lb4niNAyFYB4pRZE+ygSwf+L4qnHmONQNA9D0TIU5ETw6qtGgwJR6Yli3Nkm35N1sftdpHEcZMp3r7p1ud0rb8ofZsu3Sf68Dad/AQ==</diagram></mxfile>
```
Comment (NChitty) [inline images/Untitled Diagram.drawio]: What is this?

**Review by NChitty — CHANGES_REQUESTED** [review]

**Review by NChitty — APPROVED** [review]

**Pull request merged**